src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -292,12 +298,13 @@ class FilterPopup extends React.Component {
         Cancel
       </button>
     );
-    const resetButton = (
+    const resetButton = (ref = '') => (
       <button
         type="button"
         name="Clear-Filters"
```

Comment:
Is title case normal for name attributes like this?

Reply:
Yes, that should change to lowercase. We don't really have a convention for this at the moment.